hivemq-edge/src/frontend/src/extensions/datahub/designer/operation/OperationPanel.tsx

```diff
@@ -31,23 +35,53 @@ export const OperationPanel: FC<PanelProps> = ({ selectedNode, onFormSubmit }) =
     return adapterNode?.data
   }, [edges, nodes, selectedNode])
 
+  const pipelineIds = useMemo(() => {
+    const operationNode = nodes.find((e) => e.id === selectedNode) as Node<OperationData> | undefined
```

Comment:
suggestion(non-blocking): small suggestion for better readability

```suggestion
    const operationNode = nodes.find((node) => node.id === selectedNode) as Node<OperationData> | undefined
```